crates/phonon/src/audio_buffer.rs

```diff
@@ -37,6 +37,26 @@ impl AudioSettings {
     }
 }
 
+/// Phonon processes audio in audio buffers, which contain uncompressed Pulse
+/// Code Modulated (PCM) data (just like a .wav file).
+///
+/// Audio buffers contain one or more channels; for example, a stereo audio
+/// buffer contains 2 channels. Each channel typically contains several samples,
+/// which are values of the audio signal’s level at discrete points of time.
+/// Each channel has the same number of samples.
+///
+/// A sample is a 32-bit floating-point. The time interval between two
+/// successive samples is specified using the sampling rate. Typical sampling
+/// rates are 44100 Hz (CD quality) or 48000 Hz.
+///
+/// # Interleaving
+/// Phonon stores audio buffers in a deinterleaved layout. This means that all
+/// the samples for the first channel are store contiguously, followed by all
+/// the samples for the second channel, and so on.
+///
+/// Most audio formats instead use an interlveaved layout, where data for each
+/// frame is stored together in memory. Interleaved data can be read to and
+/// written from using [`AudioBuffer::read`] and [`AudioBuffer::write`].
```

Comment:
Might need a line break but:
```suggestion
/// written from using [`AudioBuffer::read_interleaved`] and [`AudioBuffer::write_interleaved`].
```